pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1624,7 +1599,9 @@ func configToStatus(ctx *domainContext, config types.DomainConfig,
 		ds.Vdev = xv
 
 		target := ""
-		if ds.Format != zconfig.Format_CONTAINER && !dc.ReadOnly {
+		if ds.Format == zconfig.Format_CONTAINER {
+			ds.FSVolumeLocation = getContainerPath(uuid.NewV4().String())
```

Comment:
Why do we need a new UUID? We have both a UUID for the app instance and a UUID for the image, and I think the app UUID should be sufficient for this.